frontend/src/metabase-lib/aggregation.ts

```diff
@@ -35,6 +36,22 @@ export function aggregate(
   return ML.aggregate(query, stageIndex, clause);
 }
 
+export function aggregateByCount(query: Query): Query {
+  const stageIndex = -1;
+  const operators = availableAggregationOperators(query, stageIndex);
+  const countOperator = operators.find(operator => {
+    const info = displayInfo(query, stageIndex, operator);
+    return info.shortName === "count";
+  });
+
+  if (!countOperator) {
+    return query;
+  }
+
+  const aggregation = aggregationClause(countOperator);
```

Comment:
idk do we need to use wrappers or MLv2 functions directly

Reply:
It's better to use wrappers because of the typing.
1 extra call on the stack should not be a problem :)